cellxgene_schema_cli/requirements.txt

```diff
@@ -1,5 +1,5 @@
 anndata==0.11.2
-cellxgene-ontology-guide==1.4.2 # update before a schema migration
+cellxgene-ontology-guide==1.5.0a0 # update before a schema migration
```

Comment:
this is how the alpha release of COG is tagged. you can double check this in the release history on pypi: https://pypi.org/project/cellxgene-ontology-guide/#history